src/services/fhirService.ts

```diff
@@ -73,9 +73,18 @@ interface IRecord {
   updatedDate?: Date;
   partner?: string;
 }
+
+interface AppData {
+  id?: string;
+  data: any;
+  annotations?: string[];
+  customCreationDate?: Date;
+  updatedDate?: Date;
+  partner?: string;
+}
 interface IFetchResponse {
   totalCount: number;
-  records: IRecord[];
+  records: IRecord[] | AppData[];
```

Comment:
Does AppData belong in the FHIRService?

Reply:
Good point, we should fix this together with the "I"-prefix issue in the corresponding ticket.